.github/workflows/build.yml

```
@@ -50,6 +57,9 @@ jobs:
 
     runs-on: ${{ matrix.os.runner }}
 
+    env:
+        MACOS_SIGNING: ${{ secrets.MACOS_SIGNING_IDENTITY != '' }}
```

nit: might be better to include the os preset check in here as well? then the `if` clauses in the following changes could check just one master variable. thoughts?
```suggestion
        MACOS_SIGNING: ${{ matrix.os.preset == 'mac' && secrets.MACOS_SIGNING_IDENTITY != '' }}
```

---

We can't lump them together because sometimes we need to build for non-Mac, build for the Mac but not sign, and build for the Mac and sign.  If we combined the variables, then we couldn't sometimes build for the Mac but not sign or sometimes build for the Mac and sign.

---

It would just be `MACOS_SIGNING=false` in those cases, wouldn't it? I'm just noticing that all three times `MACOS_SIGNING` is referenced, it's also and'ed with the `os.preset == 'mac'` check. If we move the `os.preset == 'mac'` check into the `MACOS_SIGNING` variable, we can remove it from the three usage sites and prevent mixups in the future if someone alters this and forgets to check it.

---

Oh, I see.  Hmmm.  But even with this change we would continue to use `if: ${{ matrix.os.preset == 'mac' }}` like when we call `Install macOS Rosetta 2` and a couple others places, right?

---

Fixed.